runtime/jcl/common/jclvm.c

```diff
@@ -300,7 +448,68 @@ hasConstructor(J9VMThread *vmThread, J9StackWalkState *state)
 	}
 }
 
+static J9HashTable *
+collectHeapStatistics(J9VMThread *vmThread) {
+	J9JavaVM *vm = vmThread->javaVM;
+	J9HashTable *hashTable = hashTableNew(
```

Comment:
The result needs to be checked for failure.

Reply:
Fixed.